probePlugin/src/main/scala/org/virtuslab/ideprobe/handlers/Projects.scala

```diff
@@ -118,6 +119,12 @@ object Projects extends IntelliJApi {
       }
   }
 
+  def setCompilerOutput(projectRef: ProjectRef, path: Path): Unit = {
+    val project = Projects.resolve(projectRef)
+    val compilerProjectExtension = CompilerProjectExtension.getInstance(project)
+    compilerProjectExtension.setCompilerOutputUrl(path.toString)
```

Comment:
shouldn't you convert Path to URL and then toString?

Reply:
Yes, but it [happens to work anyway](https://github.com/JetBrains/intellij-community/blob/46306f2ace10a4b8e5b5416d95e3c6c9831a29fc/platform/core-api/src/com/intellij/openapi/vfs/VirtualFileManager.java#L158-L162).